models/cart/src/cart/cart-draft/presets/sample-data-goodstore/jennifer-jones-01.ts

```diff
@@ -0,0 +1,63 @@
+import { AddressDraft, KeyReference } from '@commercetools-test-data/commons';
+import {
+  CustomerDraft,
+  type TCustomerDraft,
+} from '@commercetools-test-data/customer';
+import {
+  ProductVariantDraft,
+  type TProductVariantDraft,
+} from '@commercetools-test-data/product';
+import {
+  ShippingMethodDraft,
+  type TShippingMethodDraft,
+} from '@commercetools-test-data/shipping-method';
+import { StoreDraft, type TStoreDraft } from '@commercetools-test-data/store';
+import { CartDraft, LineItemDraft } from '../../../../index';
+import { origin } from '../../../constants';
+import type { TCartDraftBuilder } from '../../../types';
+
+const customer = CustomerDraft.presets.sampleDataGoodStore
+  .jenniferJones()
+  .build<TCustomerDraft>();
+const address = AddressDraft.presets.sampleDataGoodStore.jenniferJonesAddress();
```

Comment:
Question for other reviewers: should we also build the necessary address draft with the build method to ensure that it is type-safe? `AddressDraft.presets.sampleDataGoodStore.jenniferJonesAddress().build<TAddressDraft>();`

Reply:
This is reasonable to me!